src/hyper.erl

```diff
@@ -330,7 +330,7 @@ perf_report() ->
 
     R = [begin
              io:format("."),
-             random:seed(1, 2, 3),
+             rand:seed(1, 2, 3),
```

Comment:
`rand:seed/3` is undefined

20> rand:seed(1,2,3).
** exception error: undefined function rand:seed/3

Reply:
wups missed this, didn't realize they changed the new api with `rand`